src/MoBi.Core/Commands/BuildingBlockChangeCommandBase.cs

```diff
@@ -23,7 +25,7 @@ protected override void ExecuteWith(IMoBiContext context)
       {
          if (_buildingBlock == null) return;
          var buildingBlockVersionUpdater = context.Resolve<IBuildingBlockVersionUpdater>();
-         buildingBlockVersionUpdater.UpdateBuildingBlockVersion(_buildingBlock, ShouldIncrementVersion);
+         buildingBlockVersionUpdater.UpdateBuildingBlockVersion(_buildingBlock, ShouldIncrementVersion, ConversionOption);
```

Comment:
I think the behaviour ist still not fully correct.
E.g. if I have a command with `ConversionOption=SetAsExtensionModule` and apply it to a module which is already an ExtensionModule - nothing will change, so the inverse command should become `ConversionOption=None` - but it will become `ConversionOption=SetAsPKSimModule`

`buildingBlockVersionUpdater.UpdateBuildingBlockVersion` should return if the module state was changed or not.
And if not: current command should become `ConversionOption=None`